packages/facility-server/app/routes/apiv1/patient/patientProgramRegistration/patientProgramRegistrationConditions.js

```diff
@@ -1,54 +1,22 @@
 import express from 'express';
 import asyncHandler from 'express-async-handler';
 import { subject } from '@casl/ability';
-import { NotFoundError, ValidationError } from '@tamanu/shared/errors';
-import { validatePatientProgramRegistrationRequest } from './utils';
+import { NotFoundError } from '@tamanu/shared/errors';
 
 export const patientProgramRegistrationConditions = express.Router();
```

Comment:
Thoughts on moving this Router to be nested under the `patientProgramRegistration` Router rather than the `patient` Router that it's currently under? I think most of the paths would remain unchanged with the exception of the `put` route which would become either `/api/patient/programRegistration/condition/:conditionId` or `/api/patient/programRegistration/:registrationId/condition/:conditionId` ( I think I have a slight preference for the later).

I feel it'd make it a bit clearer that conditions are referenced via their registration and aren't directly linked to the patient?

Reply:
yes I agree that's a good suggestion, I think that less nesting is better and easier to follow.